markdown/developers/tutorials/adding_modules_to_pipelines.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Tutorial: Adding Modules to Pipelines'
+subtitle: Quick reference of the steps needed to add a new module to a pipeline.
+---
+
+This tutorial is a quick reference overview of the steps you will normally take when adding a new module to a pipeline.
+
+Different pipelines may have different workflows, but the following steps will cover the most common aspects.
+
+1. Install module to the pipeline
+
+   ```bash
```

Comment:
```suggestion
   ```console
```

Reply:
Disagree with this. If it is not a specific console output `bash` should be used from now on. We added some more code highlighting for it (nf-core is recognized as a function for example), and an automatic copy-button appears next to `bash` code blocks. 
Same for all the other suggested changes.

Reply:
I see, so console for all console output, but bash for the rest

Reply:
now I am wondering where would be a good place to document this?

Reply:
In the developers guidelines

Reply:
HA I WAS RIGHT! IN YO' FACE @maxulysse :-*

Reply:
I'm pretty sure you have some other `bash` -> `console` markdown stuff to update in your other PR then.
Just so you know ;-)